prediction_market_agent/markets/omen.py

```diff
@@ -440,11 +443,11 @@ def omen_sell_outcome_tx(
         web3=web3,
         market=market,
         approve=True,
-        from_address=from_address,
+        from_address=from_address_checksummed,
```

Comment:
bugfix?

Reply:
Only type-fix, code worked OK. But `get_transaction_count` used somewhere along the way requires a checksummed-type address.